charts/buffer/values.yaml

```diff
@@ -183,3 +183,67 @@ buffer:
 
 # cleanupAfterHandout -- Whether to publish message to Janitor after resource is handed out.
 cleanupAfterHandout: false
+
+liquibase-migration:
+  # In an env-specific file use something like the following to fully enable:
+  # ```
+  # liquibase-migration:   
+  #   defaults:
+  #     enabled: true
+  # ```
+  # See https://github.com/broadinstitute/terra-helm/blob/master/charts/liquibase-migration for more information
+
+  migrationJobs:
+  - name: buffer
+    # on filesystem, on classpath via app/resources/
+    migrationArgsConfigChangelog: "db/changelog.xml"
+    migrationDatabaseCredentials:
+      fromKubernetesSecret:
+        name: "buffer-postgres-db-creds"
+  - name: buffer-stairway
+    # inside stairway jar, on classpath via app/libs/*
+    migrationArgsConfigChangelog: "stairway/db/changelog.xml"
```

Comment:
@ddietterich checking Stairway DB migration path

Reply:
The path is hardcoded [here](https://github.com/DataBiosphere/stairway/blob/90755b0c79104a2d193487a89687ac8255487bf7/stairway/src/main/java/bio/terra/stairway/impl/StairwayImpl.java#L152), from what I could find. When I ran this in dev it was able to find the changelogs and acknowledged that no changes were required to the stairway db it was connected to.

Reply:
Ah, I see. I tagged Dan because I know he is making some changes on Stairway, and make it imported from TCL(terra-common-lib): https://github.com/DataBiosphere/terra-resource-buffer/pull/189
But I guess that part remains unchanged.